src/Chainweb/Chainweb.hs

```diff
@@ -230,6 +232,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configThrottleRate = 1000
     , _configMempoolP2p = defaultEnableConfig defaultMempoolP2pConfig
     , _configPruneChainDatabase = True
+    , _configBlockGasLimit = 100000
```

Comment:
Is this default still appropriate, given that gas values went up for many primitives?

Reply:
We need to set it so a full block takes less than a half second to process, that's the target I think.